### PR TITLE
DFN_QFN_LGA_SON: add TQFN-16-1EP_5x5mm_P0.8mm_EP3.1x3.1mm

### PR DESCRIPTION
**Pull request by zonque**
Used by MAXIM, as described here:

  https://pdfserv.maximintegrated.com/package_dwgs/21-0140.PDF

### CODE REVIEW

**Comment by zonque** [comment]
Footprint PR: https://github.com/KiCad/kicad-footprints/pull/1512

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/tqfn.yaml [hunk 1]
```diff
@@ -38,6 +38,51 @@ TQFN-16-1EP_3x3mm_P0.5mm_EP1.23x1.23mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+TQFN-16-1EP_5x5mm_P0.8mm_EP3.1x3.1mm:
+  device_type: 'TQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0140.PDF'
+  #custom_name_format: 'TQFN-24-1EP_4x4mm_P0.5mm_EP2.8x2.8mm_PullBack{vias:s}'
+  ipc_class: 'qfn_pull_back'
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/tqfn.yaml]: I am not so sure this is the correct IPC class for this. The "leads" extend to the edge of the package so normal qfn class should be used. (pull back is for packages that have the leads only under the part not extending to the edge. Pull back does not create space for a solder fillet on the outside.)

Comment (poeschlr) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/tqfn.yaml]: Solution: add # in front of the ipc class parameter to comment it out -> default is normal qfn class.

Comment (zonque) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/tqfn.yaml]: Thanks for the review. Updated now.

**Comment by poeschlr** [comment]
@pointhi the footprint is merged, this is good to go.

**Pull request merged**